tools/ios/MASTG-TOOL-0047.md

```diff
@@ -2,6 +2,8 @@
 title: Cydia
 platform: ios
 source: https://cydia.saurik.com/
+status: deprecated
```

Comment:
But it's still in use and working, right? Why not write a warning note in the content for now, until it's fully decommissioned? Otherwise, I think it could be confusing.

Reply:
It's _barely_ working, and only on iOS < 15 I think (or even older. Wikipedia only mentions until iOS 11). It's never going to be decommissioned, since it's just a webview on top of apt, but it's really not relevant anymore.

Reply:
I updated the phrasing